src/applications/edu-benefits/10215/containers/ConfirmationPage.jsx

```diff
@@ -1,69 +1,115 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { format, isValid } from 'date-fns';
-import { connect } from 'react-redux';
+import { connect, useSelector } from 'react-redux';
 
-import scrollToTop from 'platform/utilities/ui/scrollToTop';
-import { focusElement } from 'platform/utilities/ui';
+import { ConfirmationView } from 'platform/forms-system/src/js/components/ConfirmationView';
 
-export class ConfirmationPage extends React.Component {
-  componentDidMount() {
-    focusElement('h2');
-    scrollToTop('topScrollElement');
-  }
-
-  render() {
-    const { form } = this.props;
-    const { formId, data, submission } = form;
-    const submitDate = new Date(submission?.timestamp);
-
-    const { fullName } = data;
-
-    return (
-      <div>
-        <div className="print-only">
-          <img
-            src="https://www.va.gov/img/design/logo/logo-black-and-white.png"
-            alt="VA logo"
-            width="300"
-          />
-          <h2>Application for Mock Form</h2>
+const childContent = (
+  <div>
+    <h1 data-testid="form-title">
+      Request exemption from the 85/15 Rule reporting requirements
+    </h1>
+    <p>
+      35% exemption request from 85/15 Rule reporting requirements (VA Form
+      22-10216)
+    </p>
+    <va-alert close-btn-aria-label="Close notification" status="into" visible>
+      <h2 slot="headline">Complete all submission steps</h2>
+      <p className="vads-u-margin-y--0">
+        This form requires additional steps for successful submission. Follow
+        the instructions below carefully to ensure your form is submitted
+        correctly.
+      </p>
+    </va-alert>
+    <h2 className="vads-u-font-size--h3 vad-u-margin-top--0">
+      To submit your form, follow the steps below
+    </h2>
+    <va-process-list uswds>
+      <va-process-list-item header="Check your eligibility">
```

Comment:
All of the headers shown in the screenshot say "Check your eligibility" vs the titles shown in design

Reply:
Changing now

Reply:
Done